devices/casa_cmts.py

```diff
@@ -446,6 +447,13 @@ def check_docsis_mac_ip_provisioning_mode(self, index):
         if self.match != None:
             return result
 
+    def check_cpe_mode(self, mac):
```

Comment:
Is this really checking a mode or getting ipv4 and ipv6 addresses?

Reply:
Can we just make two new functions for this? get_cpe_ipv4_addr? get_cpe_ipv6_addr?